src/semidiscretization/semidiscretization_hyperbolic.jl

```diff
@@ -83,15 +83,15 @@ end
 # allow passing named tuples of BCs constructed in an arbitrary order
 digest_boundary_conditions(boundary_conditions, mesh, solver, cache) = boundary_conditions
 
-function digest_boundary_conditions(boundary_conditions::NamedTuple{Keys,ValueTypes}, mesh, solver, cache) where {Keys, ValueTypes<:NTuple{2,Any}} # 1D
+function digest_boundary_conditions(boundary_conditions::NamedTuple{Keys,ValueTypes}, mesh::AbstractMesh{1}, solver, cache) where {Keys, ValueTypes<:NTuple{2,Any}} # 1D
```

Comment:
Can these be fully specialized to `TreeMesh{Dim}`?

Reply:
Should work, I think :+1: